<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SalemanInfoDlg</class>
 <widget class="QDialog" name="SalemanInfoDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SalemanInfoDlg</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="TitleBarWidget" name="titleWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>45</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>45</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget_2 {
	border-image: url(:/shareImages/dlg_background.png);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>14</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>23</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget#widget_3 {
	background-color: rgb(255, 255, 255);
}</string>
        </property>
        <widget class="SalemanListView" name="listView">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>30</y>
           <width>306</width>
           <height>116</height>
          </rect>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">border:0px;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>54</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-family:&quot;微软雅黑&quot;;
font:14px;
color: rgb(51, 51, 51);</string>
         </property>
         <property name="text">
          <string>请选择：</string>
         </property>
        </widget>
        <widget class="QPushButton" name="upPageBtn">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>150</y>
           <width>87</width>
           <height>37</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>87</width>
           <height>37</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>87</width>
           <height>37</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	border:none;
	border-image: url(:/saleImage/btn-up-nomal.png);
}

QPushButton:pressed
{
	border-image: url(:/saleImage/btn-up-pess.png);
}

QPushButton:disabled
{
	border-image: url(:/saleImage/btn-up-disable.png);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="downPageBtn">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>150</y>
           <width>87</width>
           <height>37</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>87</width>
           <height>37</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>87</width>
           <height>37</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	border:none;
	border-image: url(:/saleImage/btn-down-nomal.png);
}

QPushButton:pressed
{
	border-image: url(:/saleImage/btn-down-pess.png);
}

QPushButton:disabled
{
	border-image: url(:/saleImage/btn-down-disabled.png);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="Line" name="line">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>159</y>
           <width>2</width>
           <height>21</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TitleBarWidget</class>
   <extends>QWidget</extends>
   <header location="global">TitleBarWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SalemanListView</class>
   <extends>QListView</extends>
   <header location="global">SalemanListView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
